docs/installation/additional-settings-for-developers/index.md

```diff
@@ -73,7 +111,7 @@ sudo sysctl -w net.core.rmem_max=2147483647
 
 Refer to [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html) for more information. Reading user guide for chosen DDS is helpful for more understanding.
```

Comment:
Although it's not edited in this PR, but to be consistent styles.

```suggestion
For more information, Refer to [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html). Reading user guide for chosen DDS is helpful for more understanding.
```

Reply:
Fixed at https://github.com/autowarefoundation/autoware-documentation/pull/306/commits/cebfd3c8f86dd3b92ebc9f1607a148f972c35a2b :+1: